firmware/common/adsb.hpp

```diff
@@ -83,6 +83,10 @@ const float adsb_lat_lut[58] = {
     86.53536998,    87.00000000
 };
 
+const float PI = 3.14159265358979323846;
```

Comment:
we have pi defined in other 2 places :) maybe a future cleanup will be just to use math.pi 😁

Reply:
People love PI :pie: 

But yes, at the time I didn't want to be importing another lib, but sure this needs to be cleaned up.

Thanks